go/master/python/setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, Distribution
```

Comment:
Thinking about it again, I am planning to put it together within paddle Python package, integrated with paddle's CMake.